apps/server/src/shared/infra/oauth-provider/hydra/hydra.service.ts

```diff
@@ -33,8 +33,14 @@ export class HydraService extends OauthProviderService {
 		throw new NotImplementedException();
 	}
 
-	acceptLogoutRequest(challenge: string): Promise<RedirectResponse> {
-		throw new NotImplementedException();
+	async acceptLogoutRequest(challenge: string): Promise<RedirectResponse> {
+		const hydraUri: string = Configuration.get('HYDRA_URI') as string;
+		const url = `${hydraUri}/oauth2/auth/requests/logout/accept?logout_challenge=${challenge}`;
+		const responseObservable: Observable<AxiosResponse<RedirectResponse>> = this.httpService.put(url, null, {
```

Comment:
encapsulate this in a private method to make it a bit more readable and reusable

Reply:
done